src/pip/_internal/commands/show.py

```diff
@@ -60,6 +60,10 @@ def search_packages_info(query):
         installed[canonicalize_name(p.project_name)] = p
 
     query_names = [canonicalize_name(name) for name in query]
+    missing = [name for name, pkg in zip(
```

Comment:
I think it would be good if these names were `sorted()`, for reproducibility and also readability.

Reply:
Nice idea, done!